docs/ops/deployment/yarn_setup.md

```diff
@@ -96,8 +96,6 @@ This command will show you the following overview:
 
 {% highlight bash %}
 Usage:
-   Required
-     -n,--container <arg>   Number of YARN container to allocate (=Number of Task Managers)
```

Comment:
This listing was generated by the CLI; removing it doesn't make sense if the next guy could just be adding it back.

Please change the Option description in the `FlinkYarnSessionCli` to explicitly refer to this as a deprecated legacy option, and update this section with the CLI help output..